File: LES_SERIE_S1_IA_FS_ALGO_PYTHON_C/Série_de_TD-TP_1/exercice11.c
#include <stdio.h>
#include <math.h>
float X1,X2,Y1,Y2,D;
int main(){
    printf("saisir les coordonnées de A: \n");
    printf("saisir X1: ");
    scanf("%f",&X1);
    printf("saisir Y1: ");
    scanf("%f",&Y1);
    printf("saisir les coordonnées de B: \n");
    printf("saisir X2: ");
    scanf("%f",&X2);
    printf("saisir Y2: ");
    scanf("%f",&Y2);
    D=sqrt(pow((X2-X1), 2) + pow((Y2-Y1), 2));
    printf("la distance AB=%.3f\n",D);
    return 0;
}

File: LES_SERIE_S1_IA_FS_ALGO_PYTHON_C/Série_de_TD-TP_1/exercice11.py
import math
print("saisir les coordonnées de A:")
X1=float(input("saisir X1: "))
Y1=float(input("saisir Y1: "))
print("saisir les coordonnées de B:")
X2=float(input("saisir X2: "))
Y2=float(input("saisir Y2: "))
D=math.sqrt((X2-X1)**2 + (Y2-Y1)**2)
print(f"la distance AB={D}")